ecl2df/common.py

```diff
@@ -324,7 +325,7 @@ def merge_zones(df, zonedict, zoneheader="ZONE", kname="K1"):
     zone_df = pd.DataFrame.from_dict(zonedict, orient="index", columns=[zoneheader])
     zone_df.index.name = "K"
     zone_df.reset_index(inplace=True)
-    return pd.merge(df, zone_df, left_on=kname, right_on="K")
+    return pd.merge(df, zone_df, how="left", left_on=kname, right_on="K")
```

Comment:
Could you have done this instead of first turning the zonedict into a dictionary? Or is it not so safe wrt missing values?
```
df[zoneheader] = df[kname].map(zonedict)
return df
```

Reply:
Thanks, this improves the code and passes all existing and new tests!